tools/oversight/elements/url-selector.js

```diff
@@ -55,28 +58,33 @@ export default class URLSelector extends HTMLElement {
           window.location.href = targetlocation.href;
         }
       });
-    } else {
-      fetch('https://rum.fastly-aem.page/domains?suggested=true', {
-        headers: {
-          accept: 'application/json',
-          authorization: `Bearer ${token}`,
-        },
-      }).then(async (resp) => {
-        if (!resp.ok) {
-          datalist.remove();
-        } else {
-          const { domains } = await resp.json();
-          domains.forEach((domain) => {
-            const option = document.createElement('option');
-            option.value = domain;
-            datalist.appendChild(option);
-          });
-        }
-      }).catch(() => {
-        datalist.remove();
-      });
     }
 
+    input.addEventListener('mouseover', () => {
```

Comment:
I assume this is not expected to work on mobile.

Reply:
yeah it seemed like the url-selector already wasn't working on mobile.. is that unexpected?